types/is-running/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for is-running 2.1
+// Project: https://github.com/nisaacson/is-running
+// Definitions by: Daniel Byrne <https://github.com/danwbyrne>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function isRunning(pid: number): boolean;
```

Comment:
It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).

Reply:
Hi @armanio123, I finally had a chance to look at this more thoroughly this morning. I'm fairly new to making typings but I have a LOT more of them to submit so I want to make sure I'm getting this right from the beginning. 

In our library, we prefer to not use `require(...)` and from the changes I made to match the template like you suggest that seems to be the only way to import it. What makes setting it up as: 
```
export function isRunning(pid: number): boolean;
```
and then importing it like:
```
import { isRunning } from 'is-running';
```
not preferred here? Will we just have to modify the import on our end when we use it?

thanks in advance!

Reply:
Not necessarily. You can still use the `import isRunning from 'is-running';` syntax, but you'll need to make sure that the compiler option `esModuleInterop` is enabled on your `tsconfig.json`. Details about the compiler options [here](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

Also to make things right, the d.ts file should use `export =` syntax instead. Check this [documentation](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#support-for-import-d-from-cjs-form-commonjs-modules-with---esmoduleinterop) for details.

This is how your d.ts file will look like:
```js
declare function isRunning(pid: number): boolean;
export = isRunning;
```

Reply:
additionally should I make a similar adjustment to: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/toposort

it was already approved but I think it is in the same situation as this is.

Reply:
Send me the PR. I'll be happy to check it out.

Reply:
@armanio123 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28399